lua/cli_flash.lua

```diff
@@ -92,84 +94,84 @@ lua cli_flash.lua erase -b 0 -l 4
 
 
 
-function printf(...) print(string.format(...)) end
+local function printf(...) print(string.format(...)) end
 
 --------------------------------------------------------------------------
 -- handle command line arguments
 --------------------------------------------------------------------------
 
-MODE_FLASH = 0
-MODE_READ = 2
-MODE_VERIFY = 3
-MODE_ERASE = 4
-MODE_HASH = 5
-MODE_DETECT = 6
-MODE_VERIFY_HASH = 7
-MODE_INFO = 8
-MODE_HELP = 10
-MODE_LIST_INTERFACES = 15
-MODE_DETECT_CHIPTYPE = 16
-MODE_VERSION = 17
-MODE_RESET = 18
-MODE_IDENTIFY = 19
+local MODE_FLASH = 0
```

Comment:
Actually, most of these mode definitions are no longer used since the command line parsing was rewritten to use argparse. There is a bit of clutter in cli_flash.lua that could be removed.